defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+docker_engine_version: 17.03.1.ce-1
+docker_engine_package: docker-engine-{{ docker_engine_version }}
```

Comment:
possibility for a "latest" construct? that way if you just want whatever is latest, you don't have to modify the role/variable and know what the version is. e.g. psuedo code `if docker_engine_version == latest; then docker_engine_package = docker-engine else docker_engine_package = docker-engine-{{docker_engine_version}}`